test/integration/acceptance/create_asset_test.cpp

```diff
@@ -0,0 +1,258 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <limits>
+
+#include <gtest/gtest.h>
+#include "framework/integration_framework/integration_test_framework.hpp"
+#include "framework/specified_visitor.hpp"
+#include "integration/acceptance/acceptance_fixture.hpp"
+
+using namespace integration_framework;
+using namespace shared_model;
+
+class CreateAssetFixture : public AcceptanceFixture {
+ public:
+  auto makeUserWithPerms(const interface::RolePermissionSet &perms = {
+                             interface::permissions::Role::kCreateAsset}) {
+    return AcceptanceFixture::makeUserWithPerms(perms);
+  }
+
+  const interface::types::AssetNameType kAssetName = "newcoin";
+  const interface::types::PrecisionType kPrecision = 1;
+  const interface::types::PrecisionType kNonDefaultPrecision = kPrecision + 17;
+  const interface::types::DomainIdType kNonExistingDomain = "nonexisting";
+  const std::vector<interface::types::AssetNameType> kIllegalAssetNames = {
```

Comment:
Vectors can be moved to acceptance fixture.

Reply:
Thanks!